### DISCUSSION TITLE
Make MiniHakkero visually less obtrusive

### DISCUSSION THREAD

**Discussion by erasels** [scarf005/Marisa#169] [Ideas]
Hey, Mini Hakkero is a relics that triggers on every card play and flashes itself above the player as well as flashing the Charge-up image (which lacks a big image and thus looks pretty blurry/bad) which is an extreme amount of visual noise for no real reason.

It'd be nice if you could remove the RelicAboveCreatureAction from it and maybe also make charge-up increment silently instead of flashing itself.

**Comment by scarf005** [comment]
thank you for the idea! fixed in #173.

**Comment by erasels** [comment]
Thank you! Glad to hear it, one of the biggest thing that bugged me since she had such a great looking character model.

**Reply by scarf005** [reply]
glad you liked it. please note, however, it may take more than a week until next release due to critical bug found in recent version.
